.github/dependabot.yml

```diff
@@ -0,0 +1,11 @@
+version: 2
+updates:
+- package-ecosystem: gradle
+  directory: "/"
+  schedule:
+    interval: daily
+  open-pull-requests-limit: 3
+  ignore:
+  - dependency-name: org.codehaus.groovy:groovy-all
+    versions:
+    - "> 2.4.12"
```

Comment:
@nre-ableton we might want to add a comment here to reflect what you wrote in https://github.com/AbletonDevTools/python-pipeline-utils/pull/53#issuecomment-574034577.